packages/store/src/symbols.ts

```diff
@@ -7,11 +7,14 @@ import { DispatchOutsideZoneNgxsExecutionStrategy } from './execution/dispatch-o
 
 export const ROOT_STATE_TOKEN = new InjectionToken<any>('ROOT_STATE_TOKEN');
 export const FEATURE_STATE_TOKEN = new InjectionToken<any>('FEATURE_STATE_TOKEN');
+export const NGXS_PLUGINS = new InjectionToken('NGXS_PLUGINS');
+
 export const META_KEY = 'NGXS_META';
 export const META_OPTIONS_KEY = 'NGXS_OPTIONS_META';
 export const SELECTOR_META_KEY = 'NGXS_SELECTOR_META';
 
-export const NGXS_PLUGINS = new InjectionToken('NGXS_PLUGINS');
+export type NgxsModuleOptions = Partial<NgxsConfig>;
```

Comment:
The `NgxsModuleOptions` does not need to move here. It is best kept close to it's only usage in `NgxsModule`.

Reply:
It does not need to be exposed publically.

Reply:
Why?

Reply:
done